### PR TITLE
Bug 1197489 - Fix bookmark order not being saved. r=gmarty

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=97c3cbe2ddfb7f42f1840ec679d3ac3d034d944b

File: apps/homescreen/test/marionette/bookmark_order_test.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      el.scrollIntoView();
+    });
+
+    var location1 = icons[numIcons].location();
+    var location2 = icons[numIcons-1].location();
+    actions.wait(0.5).press(icons[numIcons]).wait(0.5).
+      move(icons[numIcons-1]).release().wait(0.5).perform();
+
+    assert.equal(icons[numIcons].location().x, location2.x);
+    assert.equal(icons[numIcons-1].location().x, location1.x);
```
Comment (gmarty) [inline apps/homescreen/test/marionette/bookmark_order_test.js]: nit: `numIcons-1` should be spaced like this `numIcons - 1`.

File: apps/homescreen/test/marionette/bookmark_order_test.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  setup(function() {
+    actions = client.loader.getActions();
+    bookmark = new Bookmark(client, server);
+    home = client.loader.getAppClass('homescreen');
+    system = client.loader.getAppClass('system');
+    system.waitForFullyLoaded();
+    home.waitForLaunch();
+  });
+
+  test('Bookmark order is retained after restart', function() {
```
Comment (gmarty) [inline apps/homescreen/test/marionette/bookmark_order_test.js]: nit: It's not clear to me what's the difference between both tests. They have the same description. Can you add comments to highlight the differences and maybe change the description slightly?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5d578878dd3c292c532216a465e5d48bb0d2f85c

**Pull request merged**